contents/docs/cdp/_snippets/no-new-apps.mdx

```diff
@@ -0,0 +1 @@
+> 🚧 **Note:** We are currently **not reviewing new apps** while we develop our new [batch export](https://github.com/PostHog/posthog/issues/15997) and [webhook](https://github.com/PostHog/posthog/issues/16976) functionalities. Support their development and follow along on the linked issues. 
```

Comment:
This is nor really true now. We're fine having new apps - they must be `composeWebhook` or stateless `processEvent`